core/src/main/java/forklift/classloader/ChildFirstClassLoader.java

```diff
@@ -17,57 +18,59 @@ public ChildFirstClassLoader(URL[] classpath, ClassLoader parent) {
         system = getSystemClassLoader();
     }
 
-	@Override
-	protected synchronized Class<?> loadClass(String name, boolean resolve)
-	  throws ClassNotFoundException {
-		// First, check if the class has already been loaded
-		Class<?> c = findLoadedClass(name);
+    @Override
+    protected synchronized Class<?> loadClass(String name, boolean resolve)
+      throws ClassNotFoundException {
+        // First, check if the class has already been loaded
+        Class<?> c = findLoadedClass(name);
 
         // There are some classes that we can't go to the child first to resolve. In these instances we'll intercept the call
         // and delegate it to the system classloader. All of these relate to base jre libraries that are bundled in java.
-        if (name.startsWith("java") || name.startsWith("com.sun") || name.startsWith("sun") || name.startsWith("org.xml")) {
+        List<String> things = Arrays.asList("java", "com.sun", "sun", "org.xml", "org.w3c");
+
+        if(things.stream().filter(s -> name.startsWith(s)).count() > 0){
```

Comment:
I will make the change to .anyMatch and I will let you and Paul fight it out for the var name :)